filters/laser_similarity.json

```diff
@@ -0,0 +1,31 @@
+{
+    "type": "bilingual",
+    "description": "Filter a parallel dataset using LASER.",
+    "parameters": {
+        "THRESHOLD": {
+            "type": "float",
+            "required": false,
+            "default": 0.5,
+            "help": "Minimum accepted LASER score."
+        },
+        "BATCHSIZE": {
+            "type": "int",
+            "required": false,
+            "default": 32,
+            "help": "LASER batch size"
+        },
+        "SRCLANG": {
+            "type": "str",
+            "required": true,
+            "default": null,
+            "help": "Two-letter source language code (ISO 639-1)"
```

Comment:
Hmm, I was looking for a list of "supported" languages, given that the model has "93" in the name I'd assume that referred to a limited list. But as far as I can tell there isn't a definite list!

The language parameters here are mainly used for selecting an appropriate tokeniser. If you give it a "wrong" language, it might all still work. Maybe not ideal since I'm now assuming 93 refers to the number of languages LASER was trained on… but no real reason to limit the selection with a list of `available_values` here I suppose.

Reply:
I think that  someone who'd run it on missing language could just set the language to the "closest thing available".